src/sentry/utils/sdk.py

```diff
@@ -243,7 +243,9 @@ def configure_sdk():
     internal_project_key = get_project_key()
     upstream_dsn = sdk_options.pop("dsn", None)
     sdk_options["traces_sampler"] = traces_sampler
-    sdk_options["release"] = f"backend@{sdk_options['release']}"
+    sdk_options["release"] = (
+        f"backend@{sdk_options['release']}" if "release" in sdk_options else None
+    )
```

Comment:
Adding this as @wmak was having issues because he had a custom `SENTRY_SDK_CONFIG`